app/src/main/java/in/avimarine/boatangels/activities/BoatForInspectionActivity.java

```diff
@@ -38,7 +38,8 @@ protected void onCreate(Bundle savedInstanceState) {
     ButterKnife.bind(this);
     Query query = FirebaseFirestore.getInstance()
         .collection("boats")
-//        .orderBy("lastInspectionTime")
+        .whereGreaterThan("offerPoint", 0)
```

Comment:
This means that any boat that did not offer points is not shown in the list.
This is not the use case we discussed